### PR TITLE
Make the User Guide more explicit to prevent some of the issues reported in PED

### LINKED ISSUE #195: Documentation should be more explicit
- [x] #192 - Creation sessions are not saved
- [x] #183 - Filepaths
- [x] #179 - Need to load before importing (maybe explicitly say imported from import tab or something nicer)
- [x] #178 - Commands that do not require parameters should be explicitly mentioned to avoid this
- [x] #176 - Typo
- [x] #164 - Explain relative and absolute paths

### PR DESCRIPTION
**Pull request by gary-lgy**
Close #195

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -117,7 +117,18 @@ Examples:
...
+** On MacOS and *nix systems
+*** Absolute paths start with `/` (e.g. `/home/me/Downloads/doge.jpg`).
+*** Relative paths start directly with the file/directory name (e.g. `data/images/doge.jpg`), and is resolved in the current working directory.
+** On Windows
+*** Absolute paths start with a drive letter (e.g. `C:/Users/me/Downloads`)
+*** Relative paths are the same as those on MacOS and *nix systems.
+* Note the double `/` near the prefix `p/` in the example given.
+This is because the examples in this guide use MacOS absolute file paths, which start with a `/`.
+Therefore, the first `/` is part of the argument prefix, whereas the second `/` is part of the file path.
+If you are using MacOS or *nix systems, both `/` s must be present if your path is an absolute path.
```
Comment (moziliar) [inline docs/UserGuide.adoc]:
```suggestion
If you are using MacOS or *nix systems, both `/` s must be present if your path is an absolute path. (e.g. p//Users/me/Downloads)
```

Comment (gary-lgy) [reply docs/UserGuide.adoc]: :rofl: This is explaining the example given 10 lines above

Comment (moziliar) [reply docs/UserGuide.adoc]: what i meant was to show them that "hey! you actually need to type p// for file path! It looks dumb but ya just do it!"

Comment (gary-lgy) [reply docs/UserGuide.adoc]: Do you notice that this is under the same bullet point as line 127? This is doing exactly what you suggest - explaining why they must have both slashes.

Comment (moziliar) [reply docs/UserGuide.adoc]: oops ok... LGTM

**Pull request merged**